platform/engineering/internal-tools.md

```diff
@@ -69,22 +67,22 @@ To create and use SSH keypairs on Windows, complete the following steps:
 
 1. When prompted, enter the following name for your key (keeping the default path): 
    
-   `/c/Users/user/.ssh/id_rsa_vetsgov`
+   `/c/Users/jbritt/.ssh/id_rsa_vagov` (ensuring to swap out the user for your actual user)
```

Comment:
```suggestion
   `/c/Users/<user>/.ssh/id_rsa_vagov` (ensuring to swap out <user> for your actual user)
```

I don't know if this suggestion is much better. But definitely agree on adding the comment to swap out user for actual user.

Reply:
Although now that I see the whole script I can see why you want a real username for the example